addons/website_slides/static/src/js/snippets.animation.js

```diff
@@ -0,0 +1,15 @@
+/** @odoo-module **/
+
+import publicWidget from '@web/legacy/js/public/public_widget';
+
+publicWidget.registry.WebsiteAnimate.include({
+    /**
+     * @override
+     */
+    getScrollingElement() {
```

Comment:
Ideally, this should not be needed: from 14.0 to 17.0 included, we have had the scrollbar at a different place that the natural one (forced on the #wrapwrap), and it is was a big problem with many things.

In 18.0, we finally fixed that... other apps should do the same. Could you add a comment in the documentation comment? Something like
```javascript
    /**
     * @override
     * @todo This should be avoided: the natural scrollbar of the browser should
     * always be preferred. Indeed, moving the main scroll of the page to a
     * different location causes a lot of issues. See 189a7c96e6e26825dc05c0c64
     * for more information (improvement of 18.0 for general scrolling behaviors
     * in all website pages). E.g. issue in eLearning: go to an article in full
     * screen mode, try to use the up/down arrow keys to scroll: it does not
     * work (you first have to focus the article which should not be needed as
     * it is the only main scrollable element of the page).
     */
```